src/systems/user_interface/layouts/spawn_menu.rs

```diff
@@ -79,4 +82,168 @@ pub fn spawn_menu(
                     });
                 });
         });
+
+    let columns = 2.0;
+    let rows = 7.0;
+
+    commands
+        .spawn(NodeBundle {
+            style: Style {
+                display: Display::Grid,
+                grid_template_columns: vec![GridTrack::flex(1.0), GridTrack::flex(1.0)],
+                grid_template_rows: vec![
+                    GridTrack::flex(1.0),
+                    GridTrack::flex(1.0),
+                    GridTrack::flex(1.0),
+                    GridTrack::flex(1.0),
+                    GridTrack::flex(1.0),
+                    GridTrack::flex(1.0),
+                    GridTrack::flex(1.0),
+                ],
+                width: Val::Px(icon_size * columns),
+                height: Val::Px(icon_size * rows),
+                position_type: PositionType::Absolute,
+                left: Val::Px(icon_size),
+                top: Val::Percent(0.0),
+                ..Default::default()
+            },
+            background_color: Color::srgba(0.0, 0.0, 0.0, 0.0).into(),
+            ..Default::default()
+        })
+        .insert(SpawnMenu)
+        // Support Ship
+        .with_children(|parent| {
+            parent
+                .spawn(create_starship_button_bundle(
+                    StarshipType::SupportShip,
+                    faction.player_faction,
+                ))
+                .with_children(|parent| {
+                    parent.spawn(create_starship_button_icon(
```

Comment:
There might be a way to hide groups of UI

I would then need...

- SpawnFromStarbaseButton
- SpawnFromStarshipConstructionYardButton
- SpawnFromConstructionShipButton